kf_lib_data_ingest/common/datafile_readers.py

```diff
@@ -32,5 +41,38 @@ def read_excel_file(filepath_or_buffer, **kwargs):
             )
 
     kwargs['engine'] = 'xlrd'
-    kwargs['dtype'] = object
+    kwargs['dtype'] = str
```

Comment:
Why did you change from object to str?

Reply:
I think str is more correct than object. Also Extract was attempting to set dtype to str in a commit made more recently than that line in datafile_readers.